docs/source-1.0/guides/building-models/build-config.rst

```diff
@@ -384,6 +426,48 @@ Removes shapes if they are marked with one or more specific traits.
         }
 
 
+.. _includeShapesBySelector-transform:
+
+includeShapesBySelector
+-----------------------
+
+Includes only the shapes matching the given :ref:`selector <selectors>`.
+
+.. list-table::
+    :header-rows: 1
+    :widths: 10 20 70
+
+    * - Property
+      - Type
+      - Description
+    * - selector
+      - ``string``
+      - A valid :ref:`selector <selectors>` used to include shapes
+
+.. tabs::
+
+    .. code-tab:: json
+
+        {
+            "version": "1.0",
+            "projections": {
+                "exampleProjection": {
+                    "transforms": [
+                        {
+                            "name": "includeShapesBySelector",
+                            "args": {
+                                "selector": ":is([id=smithy.example#FooService], [id=smithy.example#FooService] ~> *)"
```

Comment:
Same comment as above: a bit simpler to do this since you don't have to repeat the service ID:
```
[id=smithy.example#FooService] :is(*, ~> *)
```